uniffi/build-release-linux.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+
+
+# Remove existing files in the destination directories
+rm -f ./target/x86_64-unknown-linux-gnu/release/libanoncreds_uniffi.a || true
+rm -f ./target/x86_64-unknown-linux-gnu/release/libanoncreds_uniffi.d || true
+rm -f ./target/x86_64-unknown-linux-gnu/release/libanoncreds_uniffi.so || true
```

Comment:
Why not remove the folder completely? :D
```suggestion
rm -r ./target/x86_64-unknown-linux-gnu || true
```

Reply:
I think it also needs this `f` flag. But I will check need time